File: strings/Making Anagrams bhatiak.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.regex.*;

public class Solution {

    public static int count(String a, char letter){
        int count=0;
        for(int i=0; i<a.length(); i++){
            if (a.charAt(i) == letter)
                count += 1;
        }
        return count;
    }


    // Complete the makeAnagram function below.
    static int makeAnagram(String a, String b, int i) {
        return (i==-1)?(0):(Math.abs(count(a, (char)(97+i)) - count(b, (char)(97+i)))) + makeAnagram(a, b, i-1);
    }

    private static final Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) throws IOException {
        BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(System.getenv("OUTPUT_PATH")));

        String a = scanner.nextLine();

        String b = scanner.nextLine();

        int res = makeAnagram(a, b, 25);

        bufferedWriter.write(String.valueOf(res));
        bufferedWriter.newLine();

        bufferedWriter.close();

        scanner.close();
    }
}

File: strings/Making Anagrams bhatiak.py
#!/bin/python3

import math
import os
import random
import re
import sys

def makeAnagram(a, b, i):
    return 0 if i==-1 else abs(a.count(chr(97+i)) - b.count(chr(97+i))) + makeAnagram(a, b, i-1)

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')
    a = input()
    b = input()
    res = makeAnagram(a, b, 26)
    fptr.write(str(res) + '\n')
    fptr.close()
